research/flamby/fed_ixi/fenda/client.py

```diff
@@ -33,7 +33,9 @@ def __init__(
         loss_meter_type: LossMeterType = LossMeterType.AVERAGE,
         metric_meter_type: MetricMeterType = MetricMeterType.ACCUMULATION,
         checkpointer: Optional[TorchCheckpointer] = None,
+        type_run: str = "cos_sim",
```

Comment:
Same comment about defaulting to vanilla fenda rather than an auxiliary loss one.